dsl/src/main/scala/asobu/dsl/ControllerMethodBuilder.scala

```diff
@@ -16,54 +17,53 @@ trait ControllerMethodBuilder {
 
   object handle {
 
-    def apply[RMT, FullRepr <: HList, K <: HList, V <: HList](directive: Directive[RMT])(implicit
-      lgen: LabelledGeneric.Aux[RMT, FullRepr],
-                                                                                         keys:   Keys.Aux[FullRepr, K],
-                                                                                         values: Values.Aux[FullRepr, V],
-                                                                                         zip:    ZipWithKeys.Aux[K, V, FullRepr],
-                                                                                         align:  Align[FullRepr, FullRepr]) =
-      new handle(Extractor.empty, directive)
+    def apply[RMT, V <: HList](directive: Directive[RMT])(
+      implicit
+      combine: CombineWithInputArgs[HNil, V, RMT]
+    ) = new handle(Extractor.empty, directive)
 
-    def apply[RMT, ExtractedRepr <: HList, FullRepr <: HList, InputRepr <: HList, V <: HList, K <: HList, TempFull <: HList](extractor: Extractor[ExtractedRepr], directive: Directive[RMT])(implicit
-      lgen: LabelledGeneric.Aux[RMT, FullRepr],
-                                                                                                                                                                                             restOf:  RestOf.Aux[FullRepr, ExtractedRepr, InputRepr],
-                                                                                                                                                                                             keys:    Keys.Aux[InputRepr, K],
-                                                                                                                                                                                             values:  Values.Aux[InputRepr, V],
-                                                                                                                                                                                             zip:     ZipWithKeys.Aux[K, V, InputRepr],
-                                                                                                                                                                                             prepend: Prepend.Aux[InputRepr, ExtractedRepr, TempFull],
-                                                                                                                                                                                             align:   Align[TempFull, FullRepr]): handle[RMT, ExtractedRepr, FullRepr, InputRepr, V, K, TempFull] = new handle(extractor, directive)
+    def apply[RMT, ExtractedRepr <: HList, V <: HList](
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Type parameter 'RMT' does not conform with naming standard.](https://www.codacy.com/app/iHeartRadio/asobu/file/2260975629/issues/source?bid=3037259&fileBranchId=3178877#l25)